README.rst

```diff
@@ -713,6 +713,12 @@ Choose from the list of available rules:
 
   Fix phpdoc inline tags, make inheritdoc always inline.
 
+  Configuration options:
+
+  - ``psr5`` (``bool``): whether the fixer should be PSR-5 compliant (uses
```

Comment:
AFAIK, https://github.com/FriendsOfPHP/Sami does not support ``@inheritdoc`` but only the inline tag, because PSR-5 was not yet drafted at this time (and phpdocumentor had not yet implemented a non-inline tag for that, as they added it later to support the Symfony use-case in a cleaner way according to their phpdoc semantic).